.travis.yml

```
@@ -12,7 +12,7 @@ branches:
 before_install:
  - pip install pytest-cov
 install:
- - pip install -e .
+ - pip install -e .[equilibrator]
```

I recommend installing as package and not using symlinks. This let's you catch distribution errors.

```suggestion
 - pip install .[equilibrator]
```